sretoolbox/container/image.py

```diff
@@ -473,8 +473,13 @@ def __eq__(self, other):
         if manifest_version == 1:
             layers_key = 'fsLayers'
         else:
-            manifest_media_type = manifest['mediaType']
-            other_manifest_media_type = other_manifest['mediaType']
```

Comment:
when there's no media type, shouldn't we fallback to the layers_key = 'layers'?

Reply:
Taking a look to the manifest this is a different type of image that looks much more like manifest list but in the OCI format:

```
{'schemaVersion': 2,
 'manifests': [{'mediaType': 'application/vnd.oci.image.manifest.v1+json',
                'digest': 'sha256:097989f522413f2b2a1b7d1eeac4d49e4155085c1d634ac1399bbab9a871f481',
                'size': 665,
                'platform': {'architecture': 'amd64', 'os': 'linux'}},
               {'mediaType': 'application/vnd.oci.image.manifest.v1+json',
                'digest': 'sha256:015857ac2b5abf8a0fe7734021c1eb1f781b1bfae8e7adfac7259b6f05852281',
                'size': 665,
                'platform': {'architecture': 'arm64', 'os': 'linux'}}]}
```

So no, we cannot fallback :( 

Re-reading the spec, I think we may want to rethink how we do the comparison as it mentions that the type of image is in the `content-type` header in the response. 

_A client will distinguish a manifest list from an image manifest based on the Content-Type returned in the HTTP response._

In this case:

```
'content-type': 'application/vnd.oci.image.index.v1+json
```

I'm going to close this PR and work on a new one based on the content of the headers.